core/ui/src/main/java/com/merxury/blocker/core/ui/topBar/SelectedAppTopBar.kt

```diff
@@ -14,41 +14,28 @@
  * limitations under the License.
  */
 
-package com.merxury.blocker.feature.search.component
+package com.merxury.blocker.core.ui.topBar
```

Comment:
```suggestion
package com.merxury.blocker.core.ui.topbar
```

Reply:
> Package names are written in all lower case to avoid conflict with the names of classes or interfaces. 

https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html